webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -6,18 +6,32 @@ import { ProjectDetails } from 'e2e/models/pages/ProjectDetails';
 import { detExecSync, fullPath } from 'e2e/utils/detCLI';
 
 test.describe('Experiement List', () => {
+  let projectDetailsPage: ProjectDetails;
+
+  // close the popover with a click elsewhere
+  const closePopover = async () =>
+    await projectDetailsPage.f_experiemntList.tableActionBar.expNum.pwLocator.click();
```

Comment:
I've been liking `keyboard.press('Escape')`. I think it's probably less prone to accidentally clicking something else, and if the wrong pop-up is open it might conveniently close it anyways.

Reply:
yeah i think i need to focus before escape. i bet i can make it work

Reply:
yeah, doesnt work on ant popovers